stress/cpuset_manager.go

```diff
@@ -68,7 +69,12 @@ func (manager *cpuStressserManager) TrackInjectorCores(cgroup cgroup.Manager, us
 	// read cpuset allocated cores
 	manager.log.Infow("retrieving target cpuset allocated cores")
 
-	cpusetCores, err := cgroup.Read("cpuset", "cpuset.cpus")
+	cpusetFile := "cpuset.effective_cpus"
+	if cgroup.IsCgroupV2() {
+		cpusetFile = "cpuset.cpus.effective"
```

Comment:
I wonder if we can make this happen magically within `cgroup.Read`, I'll look into it

Reply:
That's very specific to the cpuset controller logic so I'd avoid to put that in the cgroup package. However something which could be exploited is the stats returned by the cgroups package: https://pkg.go.dev/github.com/opencontainers/runc@v1.1.4/libcontainer/cgroups#Stats

It contains stats about cpusets so maybe we can get cores from there. However it needs to make the injector process join the targeted process cgroup first to get those stats.